polaris-core/src/main/java/org/apache/polaris/core/auth/CoreAuthorizer.java

```diff
@@ -119,8 +122,8 @@
  * the expanded roles of the calling Principal hold SERVICE_MANAGE_ACCESS on the "root" catalog,
  * which translates into a cross-catalog permission.
  */
-public class PolarisAuthorizerImpl implements PolarisAuthorizer {
-  private static final Logger LOGGER = LoggerFactory.getLogger(PolarisAuthorizerImpl.class);
+public class CoreAuthorizer implements PolarisAuthorizer {
```

Comment:
Also: wondering if this shouldn't be moved to `polaris-service` as well?

Reply:
Rename suggestion SGTM.

Not sure about moving to `polaris-service`... I was, indeed, contemplating that at some point, but later changed my mind because the logic in this class is pretty fundamental to how Apache Polaris handles permissions. This class works based on the pre-built manifest, but it is not affected by difference in the communication protocol (which is specific to `polaris-service`, I guess)... but I'm fine moving if you (or others) prefer.

Reply:
> the logic in this class is pretty fundamental to how Apache Polaris handles permissions

Are you saying that it is not possible to plug a custom `PolarisAuthorizer` then? That's worrisome.

Reply:
It is certainly possible to have a custom authorizer. The "allow all" authorizer is an example of that.

I meant that the old authorizer does not need anything that is not in `polaris-core` in order to operate. That's why I kept it in core.